processing/src/test/java/io/druid/query/select/SelectQuerySpecTest.java

```diff
@@ -75,7 +75,7 @@ public void testSerializationLegacyString() throws Exception
         DefaultDimensionSpec.toSpec(Arrays.<String>asList("market", "quality")),
         Arrays.<String>asList("index"),
         null,
-        new PagingSpec(null, 3),
+        new PagingSpec(null, 3, null),
```

Comment:
I made that null intentionally for testing that it would grab the default value from the config